config/src/config/secure_backend_config.rs

```diff
@@ -211,6 +216,7 @@ vault:
                 server: "127.0.0.1:8200".to_string(),
                 ca_certificate: None,
                 token: Token::FromDisk(PathBuf::from("/token")),
+                network_timeout_ms: 5_000,
```

Comment:
same here

Reply:
The default is 30_000;  the 5_000 that shows up here is suitable for test usage only.